RuntimePermissionsWear/Wearable/src/main/res/layout/activity_main.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
Nit, I know studio does this, but the comment should really start on the next line. ^_^;;

Reply:
Wasn't fixed but I'll approve since maybe it was missed?